aiida/cmdline/commands/computer.py

```diff
@@ -46,21 +49,17 @@ def prompt_for_computer_configuration(computer):
 
         getter_name = '_get_{}_string'.format(internal_name)
         try:
-            getter = dict(inspect.getmembers(
-                computer))[getter_name]
+            getter = dict(inspect.getmembers(computer))[getter_name]
         except KeyError:
-            print >> sys.stderr, ("Internal error! "
-                                  "No {} getter defined in Computer".format(getter_name))
+            print >> sys.stderr, ("Internal error! " "No {} getter defined in Computer".format(getter_name))
```

Comment:
print with `aiida.cmdline.utils.echo` based functions. the `echo_critical` will automatically print to stderr and exit